app/views/user_mailer/send_welcome_email.html.slim

```diff
@@ -1,18 +1,21 @@
-div style='background-color: #cfdfe4; border-radius: 20px; width: 85%; margin: 0 auto;'
-  h1 style='text-align: center; font-weight: bold; padding-top: 30px;'
+div.mailer-first-content
+  h1.message-title
     = t('user_mailer.welcome_email.welcome_title')
     = @user.full_name
     | !
-  hr style='width: 90%; text-align: center;'
-  p style='margin: 20px;'
-    p style='width: 90%; margin: 5px 5%; font-size: 18px;'
+  hr.hr-line
+  p.basic-p
+    p.paragraf
```

Comment:
typo
please change to `paragraph` everywhere

Reply:
Oh sorry, this is my very stupid mistake.